apps/dashboard/src/routes/org/[slug]/pathways/+page.svelte

```diff
@@ -0,0 +1,103 @@
+<script lang="ts">
+  import { Search, Dropdown } from 'carbon-components-svelte';
+  import PrimaryButton from '$lib/components/PrimaryButton/index.svelte';
+  import { courseMetaDeta } from '$lib/components/Courses/store';
+  import { Add } from 'carbon-icons-svelte';
+  import { isMobile } from '$lib/utils/store/useMobile';
+  import { currentOrgPath, isOrgAdmin } from '$lib/utils/store/org';
+  import { t } from '$lib/utils/functions/translations';
+  import { VARIANTS } from '$lib/components/PrimaryButton/constants';
+  import IconButton from '$lib/components/IconButton/index.svelte';
+  import Grid from 'carbon-icons-svelte/lib/Grid.svelte';
+  import List from 'carbon-icons-svelte/lib/List.svelte';
+  import { onMount } from 'svelte';
+  import { goto } from '$app/navigation';
+  import NewPathwayModal from '$lib/components/Org/PathWay/NewPathwayModal.svelte';
+  import Pathway from '$lib/components/Org/PathWay/Pathway.svelte';
+
+  export let data;
+
+  let { cantFetch } = data;
+  let searchValue = '';
+  let selectedId: string = '0';
+  let collections = [1, 2, 3, 4, 5];
```

Comment:
why is this called `collections`? and why is it an array of numbers?

Reply:
this was a quick placeholder array just to mock the array length of pathways items being returned. for the UI i should have named it pathways...but the main logic was also implemented in #408